### DISCUSSION TITLE
Improve Accessibility - add a way to change the syntax highlighting colors

### DISCUSSION THREAD

**Discussion by ameknite** [extrawurst/gitui#2251] [General]
Right now if you use a light theme. It's difficult to read the code because of colors.

example:
![Screenshot 2024-05-30 at 19 58 32](https://github.com/extrawurst/gitui/assets/104745335/f4c747de-56c0-44ce-a8f5-96fca1ebac7a)

I tried to change the theme, but it only change the interface colors, not the syntax highlighting and I don't find a way in the documentation.

**Comment by extrawurst** [comment]
oh yeah this is currently not supported
